### ISSUE TITLE
Change calibration options?

### ISSUE THREAD

**Issue by LiroyvH** [JohanDegraeve/xdripswift#261]
Currently, the toggles are a bit confusing in how you're supposed to be using them. 

I noticed the updated documentation that Paul linked on FB states: "Select the calibration method you want to use: either "Single-point calibration" or "Multi-point calibration". You can make this selection in "Bluetooth" tab in the app inside your transmitter."

This isn't actually the choice how it is presented in-app, but I think it should be. Ideally, maybe just make a drop-down menu/option (much like master/follower) called "Calibration method" which defaults to "None (Libre Algorithm)" and can be changed to "Single-point (advanced)" and "Multi-point (expert)". 

This would make it fully consistent with the documentation and be a lot less confusing?

Just a thought. :)

**Comment by jonisaksson** [comment]
Yes, I think anyone who starts with xdrip will struggle with this. Took me awhile to figure it out. 
This would be a good enhancement to simply for new users

**Comment by Jon-b-m** [comment]
Yes, and save the selected option. Now it's changed back to default "Libre Algorithm" when changing sensors, unless changing the default value in code.
